examples/python/Quantinuum_circuit_submissions.py

```diff
@@ -146,190 +145,16 @@
 
 # ### Analyze Results <a class="anchor" id="analyze-results"></a>
 
-# There are multiple options for analyzing results with pytket. A few examples are highlighted here. More can be seen at [Interpreting Results](https://tket.quantinuum.com/user-manual/manual_backend.html#interpreting-results).
+# There are multiple options for analyzing results with pytket. A few examples are highlighted here. More can be seen at [Interpreting Results](https://cqcl.github.io/pytket/manual/manual_backend.html#interpreting-results).
 
 result = backend.get_result(handle)
 print(result.get_distribution())
-
 print(result.get_counts())
 
 # ### Canceling jobs <a class="anchor" id="cancel-jobs"></a>
 
 # Jobs that have been submitted can also be cancelled if needed.
 
-# backend.cancel(handle)
-
-# ## Additional Features <a class="anchor" id="additional-features"></a>
-
-# This section covers additional features available in `pytket`.
-
-# * [Extended Circuit Compilation](#circuit-compilation-extended)
-# * [Batch Submission](#batch-submission)
-# * [Parametrized Circuits](#parametrized-circuits)
-# * [Conditional Gates](#conditional-gates)
-
-# ### Extended Circuit Compilation <a class="anchor" id="circuit-compilation-extended"></a>
-
-# This section leverages the discussion in the [Circuit Compilation](#circuit-compilation) section to illsutrate how to turn TKET compilations on or off in the `process_circuit` function, specifically for options 2 and 3.
-
-# For option 2 as described in [Circuit Compilation](#circuit-compilation), suppose a user explores the results of TKET compilation passes on a circuit and finds that `optimisation_level=1` is desirable. The submission below specifies this in the `get_compiled_circuit` function with optimization level 1. Because the circuit is optimized beforehand, the TKET optimization in the H-Series stack should be turned off. The value `tket-opt-level:None` turns off TKET optimization in the H-Series stack.
-
-compiled_circuit = backend.get_compiled_circuit(circuit, optimisation_level=1)
-
-handle = backend.process_circuit(
-    compiled_circuit, n_shots=n_shots, options={"tket-opt-level": None}
-)
-print(handle)
-
-# For option 3 as described in [Circuit Compilation](#circuit-compilation), suppose a user wants to turn off all optimizations in the stack, even simple single-qubit combinations done by the H-Series compiler. This can be done by setting `optimisation_level=0` in `get_compiled_circuit` and setting `tket-opt-level:None` in the `process_circuits` function.
-
-compiled_circuit = backend.get_compiled_circuit(circuit, optimisation_level=0)
-
-handle = backend.process_circuit(
-    compiled_circuit, n_shots=n_shots, options={"tket-opt-level": None}
-)
-print(handle)
-
-# ### Batch Submission <a class="anchor" id="batch-submission"></a>
```

Comment:
There are a couple headings that don't show up nicely in the generated notebooks

You shouldn't need any tags here -its just markkdown

If you do

```
# ### Batch Submission
```

It should be fine

Reply:
I've updated all these to remove that old way of doing tags. Originally when I made the notebooks I couldn't get the tags to work but recently found it only works if you write it exactly as it is in the header, capitlizing the words too. For example if header is ### Circuit Compilation, then the tag needs to be [Circuit Compilation](#Circuit-Compilation) whereas before I was using (#circuit-compilation) and that wasn't working. I've gone through and removed old HTML tag refs and replaced refs to other ones.